OMGVA PoS/Data layer/DTOs/Tax/TaxDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace OMGVA_PoS.Data_layer.DTOs;
+
+public class TaxDTO
```

Comment:
As I see, this class is basically identical to Tax model. Couldn't Tax model be used? Or that navigational property changes something significant?

Reply:
Ideally we should not expose internal details to the client. If the entity changes we should not break our contract and if we just use `Tax` it's very easy to do just that

Removed the TaxItems because it's a deep nested object which for now (and don't know if we will need it in the future) is not needed. Here's the example that is generated by swagger gen
![image](https://github.com/user-attachments/assets/2ad43e44-a72c-437f-9854-248674134f44)

Reply:
Ok, fine then. The question arises if we really need the navigational properties. We can leave this for later. You may resolve this comment now.